core-web/libs/dotcms-scss/angular/dotcms-theme/components/form/_radiobutton.scss

```diff
@@ -1,68 +1,75 @@
 @use "variables" as *;
 
-.p-radiobutton {
-    width: 20px;
-    height: 20px;
-    border-radius: 50%;
-    transition: box-shadow 0.2s;
-}
+p-radiobutton.p-element {
+    gap: $spacing-1;
 
-.p-radiobutton .p-radiobutton-box {
-    border: 2px solid $color-palette-gray-700;
-    background: $white;
-    width: 20px;
-    height: 20px;
-    color: $black;
-    border-radius: 50%;
-    transition: background-color 0.2s, border-color 0.2s, color 0.2s, box-shadow 0.2s,
-        background-size 0.2s cubic-bezier(0.64, 0.09, 0.08, 1);
-}
+    &.ng-dirty.ng-invalid {
+        .p-radiobutton:not(.p-radiobutton-disabled) {
+            border-color: $color-alert-red;
+        }
+    }
 
-.p-radiobutton .p-radiobutton-box:not(.p-disabled).p-focus {
-    outline: 0 none;
-    outline-offset: 0;
-    box-shadow: none;
-    border: 2px solid $border-color-highlight;
-}
+    .p-radiobutton {
+        height: 24px;
+        aspect-ratio: 1/1;
+        border-radius: 50%;
+        border: 2px solid $color-palette-gray-500;
```

Comment:
I saw a lot of new fields with a `2px` border. Should we create a variable for that new standard? @fmontes

Reply:
I made a variable for this in `common.scss` we have some parts where we use the same border in this scope